ui/admin/tests/acceptance/onboarding/success-test.js

```diff
@@ -32,18 +34,24 @@ module('Acceptance | onboarding | success', function (hooks) {
   test('check the controller url is copyable', async function (assert) {
     const origin = window.location.origin;
     await visit(urls.success);
-    assert.dom('.hds-copy-snippet__text').isVisible();
+    assert.dom(selectors.COPY_BTN_TEXT).isVisible();
     assert.strictEqual(
-      find('.hds-copy-snippet__text').textContent.trim(),
+      find(selectors.COPY_BTN_TEXT).textContent.trim(),
       origin,
     );
   });
 
   test('fill the onboarding form and redirect user to target detail when done is clicked', async function (assert) {
     await visit(urls.onboarding);
-    await fillIn('[name="targetAddress"]', '0.0.0.0');
-    await fillIn('[name="targetPort"]', '22');
-    await click('[type="submit"]');
+    await fillIn(
+      selectors.FIELD_TARGET_ADDRESS,
+      selectors.FIELD_TARGET_ADDRESS_VALUE,
+    );
+    await fillIn(
+      selectors.FIELD_TARGET_PORT,
+      selectors.FIELD_TARGET_PORT_VALUE,
+    );
+    await click(commonSelectors.SAVE_BTN);
     await click(doneButtonSelector);
     const projectId = this.server.db.scopes.where({ type: 'project' })[0].id;
```

Comment:
I know this wasn't your work but I figured this can also be a learning moment. We have a history of using `this.server.db` and `this.server.schema`. We decided a while ago to move all `db` uses to `schema`. In this instance, the update would look like this (auth method example I found):
```
const authMethod = this.server.schema.authMethods.findBy({
      id: instances.authMethod.id,
    });
```